src/firecracker/src/main.rs

```diff
@@ -93,6 +93,11 @@ fn main() {
                 .takes_value(true)
                 .help("Process start CPU time (wall clock, microseconds)."),
         )
+        .arg(
+            Argument::new("jailer-time-cpu-us")
+                .takes_value(true)
+                .help("Jailer process CPU time (wall clock, microseconds)."),
+        )
```

Comment:
Could this just be hidden inside process start time? I would avoid having jailer specific logic in firecracker.

Reply:
See [comment above](https://github.com/firecracker-microvm/firecracker/pull/2488#discussion_r604873051)